instrumentation/elasticsearch/elasticsearch-api-client-7.16/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/apiclient/ElasticsearchEndpointMap.java

```diff
@@ -0,0 +1,708 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.elasticsearch.apiclient;
+
+import io.opentelemetry.javaagent.instrumentation.elasticsearch.rest.ElasticsearchEndpointDefinition;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+public class ElasticsearchEndpointMap {
+
+  private static final Map<String, ElasticsearchEndpointDefinition> routesMap = new HashMap<>(415);
```

Comment:
Nit: consider creating a temp var with that hashmap in the `static {}` block, and setting the `routesMap` to `Collections.unmodifiableMap(routes)` at the end of the block.

Reply:
done